composition-js/src/__tests__/compose.test.ts

```diff
@@ -4630,4 +4642,75 @@ describe('composition', () => {
     const authenticatedDirectiveExists = schema.directives().find(d => d.name === 'authenticated');
     expect(authenticatedDirectiveExists).toBeUndefined();
   });
+
```

Comment:
formatting is off.

Reply:
https://github.com/apollographql/federation/pull/2894/commits/e4076f039b09637097b3e314740e06dbeb27935b